ibm_catalog.json

```diff
@@ -1042,6 +1061,14 @@
                   "dependency_input": "logs_routing_tenant_regions",
                   "version_input": "logs_routing_tenant_regions",
                   "reference_version": true
+                },
+                {
+                  "version_input": "install_logging_agent",
+                  "value": true
+                },
+                {
+                  "dependency_output": "ingestion_endpoint",
```

Comment:
this is wrong and its failing the tests:
```
2025-12-07T18:12:21.7043851Z [TestAddonDefaultConfiguration] ✘   ref://project.addon-vsideft-82u/configs/deploy-arch-ibm-cloud-logs-wxqumr/outputs/ingestion_endpoint - Error: The project reference cannot be found
2025-12-07T18:12:21.7044812Z [TestAddonDefaultConfiguration]       Message: The project reference cannot be found
2025-12-07T18:12:21.7045299Z [TestAddonDefaultConfiguration]       Code: 404
```
There is no output called `ingestion_endpoint` in the Cloud Logs DA. I think the output you want is called `cloud_logs_ingress_private_endpoint`. Please double check that and verify it works. Thanks